api/grpcserver/v2alpha1/activation.go

```diff
@@ -151,17 +157,25 @@ func (s *ActivationStreamService) Stream(
 	}
 }
 
-func toAtx(atx *types.VerifiedActivationTx) *spacemeshv2alpha1.ActivationV1 {
+func toAtx(
+	ctx context.Context,
+	db sql.Executor,
+	atx *types.VerifiedActivationTx,
+) (*spacemeshv2alpha1.ActivationV1, error) {
+	positioning, err := atxs.PositioningATX(ctx, db, atx.ID())
+	if err != nil {
+		return nil, err
+	}
 	return &spacemeshv2alpha1.ActivationV1{
 		Id:             atx.ID().Bytes(),
 		Signature:      atx.Signature.Bytes(),
 		PublishEpoch:   atx.PublishEpoch.Uint32(),
 		PreviousAtx:    atx.PrevATXID[:],
-		PositioningAtx: atx.PositioningATX[:],
+		PositioningAtx: positioning.Bytes(),
```

Comment:
I think we can just drop this from the API. The Positioning ATX imo has no value to someone using the API. We can instead add a new API method that allows a user to get the raw bytes for the ATX and decode it themselves if they are interested in information that our "easy use" API doesn't provide. Wdyt?

Reply:
Fine with me, but we should design the API to best fit the required use case. If the positioning ATX is really needed there, then it should stay. If not I'd happily remove it as obtaining it complicates the code :)

@kacpersaw wdyt?

Reply:
Fine with me as well 👍